platform/ios/src/MGLMapView.mm

```diff
@@ -293,6 +293,8 @@ @implementation MGLMapView
     BOOL _accessibilityValueAnnouncementIsPending;
 
     MGLReachability *_reachability;
+    
+    BOOL _showsTraffic;
```

Comment:
Is this necessary? Doesn't the property already have a backing ivar with this name?